.github/workflows/on_pr.yml

```
@@ -70,7 +134,7 @@ jobs:
   #                 still finishing (e.g. Mergify CI auto-retry). Treat as
   #                 non-success — do not whitelist only failure/cancelled.
   all:
-    needs: [ main_workflow, upload_metrics_history ]
+    needs: [ main_workflow, upload_metrics_history, wait_for_push_schedule ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)